src/main/java/com/sumologic/client/Credentials.java

```diff
@@ -7,37 +7,62 @@
  * @author Sebastian Mies
  */
 public class Credentials {
-    private String email;
-    private String password;
+    private String accessId;
+    private String accessKey;
 
     /**
-     * Constructs the credentials using email and password.
+     * Constructs the credentials using accessId and accessKey.
      *
-     * @param email Your email
-     * @param password Your password
+     * @param accessId Your access id
+     * @param accessKey Your access key
      */
-    public Credentials(String email, String password) {
-        this.email = email;
-        this.password = password;
+    public Credentials(String accessId, String accessKey) {
```

Comment:
If I'm understanding this correctly, I think that one thing @lei-sumo forgot to mention is that we can track users on the server side from the logs as to who is authenticating with username/password vs access keys with the api. In this way, we can make sure everyone is migrated and reach out to those who have not yet migrated to using access keys.